javascript-data-model/order-history.js

```diff
@@ -0,0 +1,118 @@
+var orderHistory = [
+  {
+    dateOrderPlaced: 'August 4, 2020',
+    totalPrice: '$34.00',
+    shipTo: 'JS Masher',
+    orderNumber: '114-3941689-8772232',
+    orderDetails: '',
+    invoice: '',
+    delivered: {
+      date: 'Aug 8, 2020',
+      methodDelivered: ''
+    },
+    imagePreviews: [''],
```

Comment:
Rather than storing image previews separately, i would add an `image` property to the item itself.